### PR TITLE
Chore/switch from jest to vitest

### PR DESCRIPTION
**Pull request by work-kevin-flynn**
- switch to use vitest
- verified locally
- verified in CI

### CODE REVIEW

File: packages/did-ui-react/style.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+import './src/assets/theme/modified.antd.css';
```
Comment (FrankFeng1116) [inline packages/did-ui-react/style.ts]: index.less already import this css file, Is this job redundant?

File: packages/next-example/src/app/layout.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import type { Metadata } from 'next';
 import { Inter } from 'next/font/google';
+import '@portkey/did-ui-react/dist/assets/index.css';
```
Comment (FrankFeng1116) [inline packages/next-example/src/app/layout.tsx]: The usage has changed and needs to be add explanation in the document (portkey-doc project)

File: packages/utils/src/storageMethods.ts [hunk 3]
```diff
@@ -7,3 +7,4 @@
 //   removeItem = async (key: string) => {};
 //   setItem = async (key: string, value: any) => {};
 // }
+export default {};
```
Comment (FrankFeng1116) [inline packages/utils/src/storageMethods.ts:10]: What is the purpose of adding this line of code?

**Review by FrankFeng1116 — APPROVED** [review]
LGTM

File: packages/did-ui-react/style.ts (hunk 1 above)
Comment (work-kevin-flynn) [reply packages/did-ui-react/style.ts]: makes sense. I will move the style fix to another PR https://github.com/Portkey-Wallet/portkey-web/pull/330

File: packages/utils/src/storageMethods.ts (hunk 3 above)
Comment (work-kevin-flynn) [reply packages/utils/src/storageMethods.ts:10]: sometimes during local test runs both jest and vitest would complain that empty file has no exports so I left a default there. Maybe it should be deleted?

File: packages/next-example/src/app/layout.tsx (hunk 2 above)
Comment (work-kevin-flynn) [reply packages/next-example/src/app/layout.tsx]: yes, will do in the other PR https://github.com/Portkey-Wallet/portkey-web/pull/330

**Pull request merged**